src/smart-components/request/request-detail/request.js

```diff
@@ -23,29 +23,34 @@ import {
   Level,
   LevelItem
 } from '@patternfly/react-core';
+import UserContext from '../../../user-context';
 
-class Request extends Component {
-  state = {
-    isKebabOpen: false
+export const Request = ({ item, isExpanded, toggleExpand }) => {
+  const [ isKebabOpen, setIsKebabOpen ] = useState(false);
+  const { userPersona: userPersona } = useContext(UserContext);
+
+  const onKebabToggle = isOpen => {
+    setIsKebabOpen(isOpen);
   };
 
-  onKebabToggle = isOpen => {
-    this.setState({
-      isKebabOpen: isOpen
-    });
+  const onKebabSelect = () => {
+    setIsKebabOpen(!isKebabOpen);
```

Comment:
We should use callback for the setter to avoid potential sync issues
`setIsKebabOpen(isKebabOpen => !isKebabOpen);`